server/src/helpers/filestore-cloud.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const { S3Client, PutObjectCommand, DeleteObjectCommand } = require('@aws-sdk/client-s3'),
+	config = require('../../config/config'),
+	fsutil = require('./filestore-utils');
+
+const s3Client = new S3Client({
+	region: config.storage.region,
+	endpoint: config.storage.endpointURL,
+	forcePathStyle: true,
+	signatureVersion: 'v4',
+});
+
+module.exports = {
+
+	storeFileCloud: async (fileBuffer, fileName) => {
+		const results = await s3Client.send( new PutObjectCommand( {
+			Bucket: config.storage.bucketName,
+			Key: fileName,
```

Comment:
This should be called file key, if we want to store more things in cloud then we'll be separating them out using a folder structure-like key for it (E.g "maps/filename.extension" or "asset/atlas/filename.extension"). It's just a small thing but naming the functions args will clear that up

Reply:
Changed in latest push.